src/Features/CSharp/Portable/Completion/CompletionProviders/OverrideCompletionProvider.cs

```diff
@@ -164,23 +171,48 @@ public override bool TryDetermineReturnType(SyntaxToken startToken, SemanticMode
 
                         break;
                     default:
-                        // Anything else and we bail.
-                        return false;
+                        // If we faced modifier, that we don't accept above (e.g. `static`)
+                        // then this modifier is not valid for override completions, so we can just exit
+                        if (SyntaxFacts.IsKeywordKind(token.Kind()))
+                        {
+                            return false;
+                        }
+
+                        break;
+                }
+
+                if (SyntaxFacts.IsKeywordKind(token.Kind()) &&
+                    !IsOnStartLine(token.SpanStart, text, startLine))
+                {
+                    multilineModifiers = true;
                 }
 
                 var previousToken = token.GetPreviousToken();
 
                 // We want only want to consume modifiers
-                if (previousToken.IsKind(SyntaxKind.None) || !IsOnStartLine(previousToken.SpanStart, text, startLine))
+                if (previousToken.IsKind(SyntaxKind.None))
                 {
                     break;
                 }
 
+                lastSeenToken = token;
                 token = previousToken;
             }
 
             modifiers = new DeclarationModifiers(isUnsafe: isUnsafe, isAbstract: isAbstract, isOverride: true, isSealed: isSealed, isRequired: isRequired);
-            return overrideToken.IsKind(SyntaxKind.OverrideKeyword) && IsOnStartLine(overrideToken.Parent!.SpanStart, text, startLine);
+
+            if (!overrideToken.IsKind(SyntaxKind.OverrideKeyword) || multilineModifiers)
```

Comment:
the use of 'multiLineModifiers' seems odd to me.  It seems to exist only to set that bool when modifiers are not on the same line.  Then it is never set to false.  Then we'll fall out to this code, see that it is true and bail out.  So why do we need the flag at all.  Why not just remove it and jsut reutrn immediately when you see the modifier is on a different line in the above loop?